### ISSUE TITLE
Eliminate confusion resulting from the Continue Exploring button on the Checkout page

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/jazkarta.shop#31]
I just walked through a book purchase on the SD site in prep for describing how it works to UMP: I added a book to my cart, went to my cart, filled out shipping info, proceeded to checkout, but then (since I had no intention of purchasing) from that page I went back to my cart, removed the item and clicked Continue Exploring. I was redirected back to the Checkout page with a $0 purchase now showing, and Complete Purchase is my only option. It would have been nice to Cancel my purchase and not have to see this $0 purchase page.

**Comment by davisagli** [comment]
When you say you "went back to my cart", did you do that by clicking Back in your browser, or by clicking a link in the site? The "Continue Exploring" button just tells your browser to navigate back, so if you returned to the cart by clicking a link, that would have added an entry to your browser history, and Continue Exploring would have returned to the previous entry, which was the checkout page.

The simplest solution is probably to make Continue Exploring just go to the homepage or remove it entirely. A more complicated solution would be to keep the current behavior in most cases, but go to the homepage if the previous page (from HTTP referer) is something that we recognize as part of the checkout process.

**Comment by skleinfeldt** [comment]
To clarify: I added a book to my cart, went to my cart, filled out shipping info, proceeded to checkout, then from that page I clicked on the Cart link in the top right, removed the item and clicked Continue Exploring. 

So yes this is explained by navigating back to the previous page. And yes I think the simple solution of Continue Exploring either going to the homepage or being removed is fine - whichever is easier or seems more sensible. (The more complicated solution would be ideal, but let's keep it simple for now.)

Changed ticket title accordingly.
